src/main/kotlin/lotto/domain/LottoRank.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.domain
+
+enum class LottoRank(
+    val matchCount: Int?,
+    val price: Int
+) {
+    NONE(null, 0),
+    FOURTH(3, 5000),
+    THIRD(4, 50000),
+    SECOND(5, 1500000),
+    FIRST(6, 2000000000);
```

Comment:
코틀린에서는 숫자를 표현할 때 언더바 ( _ )로 구분해 가독성을 더 높일 수 있어요!
```suggestion
    FOURTH(3, 5_000),
    THIRD(4, 50_000),
    SECOND(5, 1_500_000),
    FIRST(6, 2_000_000_000);
```

혹여나 1등이 2개 이상 발생할 수 있으니, 상금은 Long으로 변경해보는 건 어떨까요?

Reply:
@malibinYun 제가 이해를 못했는데 1등이 2개 이상 발생하는 것과 Int를 Long으로 타입 변경이 어떤 관계가 있나용?

Reply:
값을 꺼내 더할 때에 Int인지 Long인지 파악하지 못하고 바로 더해버리는 케이스가 있을 지도 몰라 제안드렸어요!